plugins/modules/idrac_server_config_profile.py

```diff
@@ -695,26 +749,28 @@ def get_scp_share_details(module):
     share_name = module.params.get("share_name")
     command = module.params["command"]
     scp_file_name_format = get_scp_file_format(module)
+    share = {}
+    if share_name:
+        share = parse_share_name(module, share_name)
+        if command in ["export", "export_custom_defaults"]:
+            share["file_name"] = scp_file_name_format
+    return share, scp_file_name_format
+
+
+def parse_share_name(module, share_name):
     if ":/" in share_name:
         nfs_split = share_name.split(":/", 1)
-        share = {"share_ip": nfs_split[0], "share_name": "/{0}".format(nfs_split[1]), "share_type": "NFS"}
-        if command == "export":
```

Comment:
Any reason to remove this line of code, 

earlier it was adding/updating file_name key in share dictionary when the command was export.

below return doesn't contain file_name key?

Reply:
It was moved to a separate function and that function is called at line 755